src/docs/sdk/performance/modules/screen-loads.mdx

```diff
@@ -2,4 +2,26 @@
 title: 'Screen Loads Module'
 ---
 
-to be defined ...
+The Screen Loads module records metrics from spans in `ui.load` transactions using specific span operations.
+
+# Span Conventions
+
+## Span Operations
+| Span OP | Description |
+|:--|:--|
+| `db.sql.query` | A database query |
+| `db.sql.room` | A database query using the Room library on Android |
+| `db.sql.transaction` | - |
+| `db` | An operation on the database |
+| `file.read` | Reading a file from the file system |
+| `file.write` | Writing to a file on the file system |
+| `http.client` | An outgoing network request |
+| `ui.load` | An operation on a mobile UI |
+| `ui.load.full_display` | When the screen has fully completed loading |
+| `ui.load.initial_display` | When the screen has rendered its first frame |
+
+## Span Measurements
+| Key | Unit | Description |
+|:--|:--|:--|
+| `time_to_initial_display` | milliseconds | The time it took to load the first frame |
+| `time_to_full_display` | milliseconds | The time it took to fully load the screen |
```

Comment:
Are these required?

Reply:
Good question. I actually realized these are required on the _transaction_ level at the moment (some of the widgets here still query transaction metrics) so I've split it out into a similar "Transaction Conventions" section.

`time_to_initial_display` is required for the module but `time_to_full_display` is optional and we prompt the user to manually instrument it.